src/Database/Type/BoolType.php

```diff
@@ -26,30 +24,8 @@
  *
  * Use to convert bool data between PHP and the database types.
  */
-class BoolType extends Type implements TypeInterface, BatchCastingInterface
+class BoolType extends BaseType implements BatchCastingInterface
```

Comment:
Should we also remove the checka that prevent core type classes from using batch casting?

Reply:
The type check in `FieldTypeConverter` has been removed now. I believe that's what you are referring to.